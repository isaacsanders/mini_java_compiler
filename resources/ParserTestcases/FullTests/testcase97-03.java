class main extends main {
	public main main() {
		System.out.println(main);
		return !main != !main - main;
	}
	public void main(main main) {
		// I'm Corvec
		return main;
	}
	public void main(main main) {
		// I'm Corvec
	}
	main main;
	main main;
	main main;
}

class main {
	public static void main(String [] main) {
		return main
	}
}

