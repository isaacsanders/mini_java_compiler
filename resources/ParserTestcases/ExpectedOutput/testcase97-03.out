

ERROR in line 1 and column 7: Syntax error



ERROR in line 1 and column 7: Couldn't repair and continue parse

Exception processing file: Can't recover from previous error(s)
